manifests/4.5/logging.openshift.io_clusterlogforwarders_crd.yaml

```diff
@@ -112,18 +116,35 @@ spec:
                           cron authpriv ftp ntp security console solaris-cron     local0
                           local1 local2 local3 local4 local5 local6 local7"
                         type: string
+                      msgID:
+                        description: "MsgID is MSGID part of the syslog-msg header
+                          \n MsgID needs to be specified if using rfc5424"
+                        type: string
                       payloadKey:
```

Comment:
Should be `payload` to be consistent with `tag`. Need to document the $key convention.

Reply:
`payload` can only refer to a path in record, where as `tag` or `appname` can refer to path in record or fluentd tag